shell.go

```diff
@@ -113,7 +113,7 @@ func shell(args []string) error {
 		return err
 	}
 	defer func() {
-		err = multierr.Combine(err, term.RestoreTerminal(os.Stdin.Fd(), termState))
+		err = errors.Join(err, term.RestoreTerminal(os.Stdin.Fd(), termState))
```

Comment:
:exploding_head: Thanks for pointing this out! I've wondered why `multierr.Combine` isn't built into one of the standard libraries. I can't wait until the RDK is running go 1.20...

Reply:
Yeah, figured this was a good time to drop that dependency and modernize.